easybuild/easyconfigs/p/PLplot/PLplot-5.11.1-foss-2016a-Java-1.7.0_79-Python-2.7.11.eb

```diff
@@ -0,0 +1,62 @@
+easyblock = 'CMakeMake'
+
+name = 'PLplot'
+version = '5.11.1'
+
+homepage = 'http://plplot.sourceforge.net'
+description = """PLplot is a cross-platform software package for creating scientific plots whose (UTF-8) plot symbols
+ and text are limited in practice only by what Unicode-aware system fonts are installed on a user's computer."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'pic': True}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+pyver = '2.7.11'
+javaver = '1.7.0_79'
```

Comment:
@SimonPinches please bump to 1.7.0_80, and use `%(javaver)s` and `%(pyver)s`, done in https://github.com/SimonPinches/easybuild-easyconfigs/pull/6